src/relative-time.js

```diff
@@ -39,9 +40,9 @@ function startOf(date, unit) {
   return date;
 }
 
-export default class RelativeTime {
+class RelativeTime {
   constructor() {
-    this.formatters = RelativeTime.initializeFormatters(...arguments);
+    this.rtf = new Intl.RelativeTimeFormat(undefined);
```

Comment:
We may need to include the `numeric: "auto"` option here depending on how we solve https://github.com/tc39/proposal-intl-relative-time/pull/60